### ISSUE TITLE
You can no longer install BYOND versions

### ISSUE THREAD

**Issue by AffectedArc07** [tgstation/tgstation-server#1287]
**Describe the bug**
You can no longer install BYOND versions

Likely due to lummox changing the website around, as anything HTTP on the BYOND website doesnt work anymore

**To Reproduce**
1. Go to install a byond version
2. 
![image](https://user-images.githubusercontent.com/25063394/124474321-4ce86c00-dd98-11eb-8b43-3f95860bc31d.png)

**Expected behavior**
It to download properly

**Logs**
Provided on request

**Server State: (please complete the following information):**
 - OS: Windows 
 - Version: TGS 4.12.1 
 - Database Type/Version: MariaDB
 - BYOND Version Used: 514.1542
 - git Repository Used: N/A
 - Origin Commit hash Used: N/A
 - Active Test Merges: N/A
 - Client Version: 2.1.2.0

**Additional context**
N/A

**Comment by AffectedArc07** [comment]
I am a certified idiot

**Comment by Cyberboss** [comment]
TL;DR 1542 is a 513 version, not 514
